README.md

```diff
@@ -170,3 +172,52 @@ This produces
 `((c (i (= (+ 2 5) (q 10)) (q (q "the sum is 10")) (q (q "the sum is not 10"))) 1))`
 
 which is not much code, for how much source there is. This also demonstrates the general notion that macros (and inline functions) cause much less code bloat than functions. The main disadvantages is that macros are not recursive (since they run at compile time) and they're messier to write.
+
```

Comment:
The info below is great, but it probably doesn't belong here since it's about upstream dependencies (which might even change in the near future). I will copy it to the Wiki, and you can put in a link to that.

Reply:
I added https://github.com/Chia-Network/chia-blockchain/wiki/Help-with-clvm_tools

Reply:
Sure, I removed the troubleshooting section on the latest commit. Could we add the following steps to the debian 10 section of https://github.com/Chia-Network/chia-blockchain/wiki/Help-with-clvm_tools ? I think what you copied there was the readme from one of my first commits (and later more details were added).

```
This issue shows up when using 'run/brun' commands. Credit to @sgharvey on keybase for the solution:

1. Add backports to your sources.list (or alternatively add a new file with the ".list" extension to /etc/apt/sources.list.d/)
```
deb http://deb.debian.org/debian buster-backports main
```

2. Run ```apt-get update```
3. Install cmake from backports
```
apt-get install cmake/buster-backports
```
4. Clone into clvm
```
git clone https://github.com/Chia-Network/clvm.git
cd clvm
```
5. Create venv
```
python3 -m venv venv
. ./venv/bin/activate
```
7. Install clvm into venv
```
pip install -e '.[dev]'
```
8. If you pay close attention, you'll notice that clvm_tools was installed. Test that by running 'run' or 'brun'.
```

Thanks!

Reply:
Done

Reply:
Just to be clear, this is addressing a build problem with `blspy`, right?

Reply:
The debian 10 steps are related to the "No module named 'clvm.CLMObject'" error.